test/remote/gateways/remote_pay_gate_xml_test.rb

```diff
@@ -52,4 +54,13 @@ def test_invalid_login
     assert_failure response
     assert_equal 'Incorrect Credentials Supplied', response.message
   end
+
+   def test_purchase_and_full_credit
```

Comment:
Also, this remote test is doing a purchase and refund, renaming this to ``` test_successful_purchase_and_refund ``` would bring the wording in line with the expectation of the test a little more